### PR TITLE
25 avatar component

### PR DESCRIPTION
**Pull request by Roman6917**

### CODE REVIEW

**Review by rostyslav-vasyliuk — APPROVED** [review]

**Review by romanolytvyn — APPROVED** [review]

**Review by AndriyTykhan — APPROVED** [review]

**Review by Yuri-Pristay — APPROVED** [review]

File: src/components/navProfile/Avatar.jsx [hunk 1]
```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import AvatarEdit from 'react-avatar-edit';
+import Modal from 'react-modal';
```
Comment (Antoniozinchenko) [inline src/components/navProfile/Avatar.jsx]: why do you install additional lib???
sematic-ui has its own Modal component
https://react.semantic-ui.com/modules/modal/

File: src/components/navProfile/Avatar.jsx [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+  handleUpload = () => {
+    document.getElementById('avatar').style.display = 'block';
+    this.setState({
+      modalIsOpen: true,
+    });
+  };
+
+  render() {
+    console.log(this.state.preview);
```
Comment (Antoniozinchenko) [inline src/components/navProfile/Avatar.jsx]: remove console logs

File: src/components/navProfile/Avatar.jsx [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+  };
+
+  onCrop = (preview) => {
+    this.setState({
+      preview,
+    });
+  };
+
+  handleUpload = () => {
+    document.getElementById('avatar').style.display = 'block';
```
Comment (Antoniozinchenko) [inline src/components/navProfile/Avatar.jsx]: do not use getElementById
bad practice

File: src/components/navProfile/Avatar.jsx [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+    this.setState({
+      modalIsOpen: true,
+    });
+  };
+
+  render() {
+    console.log(this.state.preview);
+    console.log(this.state.src);
+    return (
+      <div style={{
```
Comment (Antoniozinchenko) [inline src/components/navProfile/Avatar.jsx]: remove inline styles

**Review by Antoniozinchenko — CHANGES_REQUESTED** [review]

**Review by Antoniozinchenko — CHANGES_REQUESTED** [review]
Squash all commits to one

File: src/components/navProfile/Avatar.jsx (hunk 1 above)
Comment (Roman6917) [reply src/components/navProfile/Avatar.jsx]: okay

File: src/components/navProfile/Avatar.jsx (hunk 2 above)
Comment (Roman6917) [reply src/components/navProfile/Avatar.jsx]: okay, i have forgotten about it

File: src/components/navProfile/Avatar.jsx (hunk 3 above)
Comment (Roman6917) [reply src/components/navProfile/Avatar.jsx]: okay

File: src/components/navProfile/Avatar.jsx (hunk 4 above)
Comment (Roman6917) [reply src/components/navProfile/Avatar.jsx]: good

**Review by Antoniozinchenko — APPROVED** [review]

**Pull request merged**